### PR TITLE
enh(UI): Better delimitation between menu and content

### PR DESCRIPTION
**Pull request by EdytaSki**
<h1> Pull Request Template </h1>

<h2> Description </h2>

Better delimitation between menu and content
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

Fixes # (issue)

<h2> Type of change </h2>

Please delete options that are not relevant.

- [ ] New functionality (non-breaking change)

<h2> Target serie </h2>

Please delete series that are not relevant.

- [ ] 18.10.x

<h2> How this pull request can be tested ? </h2>

The left menu bar should have the right grey border.

Any **relevant details** of the configuration to perform the test should be added.

<h2> Checklist </h2>

<h5> Community contributors & Centreon team </h5>

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).
- [ ] I have updated the **release note** in dedicated temporary section **\***

**\*** updating the release note results in adding the **pull request id** and **description** at the end of the file. **Product Managers** will rework it for the release. Release notes can be found [here](https://github.com/centreon/centreon/tree/master/doc/en/release_notes).

<h5> Centreon team only </h5>

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests covers 80%** of the code written for the story.
- [x] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

**Comment by sc979** [comment]
image

**Comment by sc979** [comment]
image

**Comment by EdytaSki** [comment]
The issue has been resolved with z-index property on .sidebar.

File: www/front_src/src/styles/partials/_side-nav.scss [hunk 1]
```diff
@@ -13,6 +13,7 @@
   text-align: center;
   background: #e0e0e0;
   transition: all 0.2s;
+  border-right: 1px solid rgba(209, 210, 212, 1);
```
Comment (jdelpierre) [inline www/front_src/src/styles/partials/_side-nav.scss]: You declare this variable : **$grey-medium** in __variables.scss_, but you don't use here ?

Isn't the seconde solution better that the first one ?
`border-right: 1px solid $grey-medium;

Comment (EdytaSki) [reply www/front_src/src/styles/partials/_side-nav.scss]: done

**Pull request merged**